src/metatensor/models/experimental/alchemical_model/train.py

```diff
@@ -39,7 +39,7 @@ def train(
     hypers: Dict = DEFAULT_HYPERS,
     continue_from: Optional[str] = None,
     output_dir: str = ".",
-    device_str: str = "cpu",
+    devices: List[torch.device] = [torch.device("cpu")],  # noqa: B006, B008
```

Comment:
Please don't =) B006 is very relevant here. 

This should either be `None` or not provided (I don't think we need the default values here, they will always be provided by metatensor-models, right?)

Reply:
You're right. The default values are just handy because we can call these functions from our tests with fewer arguments